documentation/Using-Azure-TestFramework.md

```diff
@@ -105,17 +106,21 @@ Changing the above properties at run-time has the potential to hard code few thi
 
 Ensure that the `HttpRecorderMode` in the `TEST_CSM_ORGID_AUTHENTICATION` environment variable is consistent with the value in `AZURE_TEST_MODE` environment variable.
 
-## 7. Environment Variable Reference
+##7. Connection string
```

Comment:
@shahabhijeet make sure there is a space between "##" and "7" so that it renders correctly in markdown.

Also, should this be "Supported Key-Value pairs in ConnectionString" so it matches the corresponding bullet in the table of contents at the top?